### PR TITLE
ger Users by organization Id

### PR DESCRIPTION
**Pull request by ghost**
By the API below 
`//ENDPOINT /api/organizations/:organization_id/users/ :get`

you can have a list of users in the same Organization!

### CODE REVIEW

File: src/server/api/controllers/user.controller.js [hunk 1]
```diff
@@ -321,6 +321,16 @@ const createAdminBySuperUser = async (id, body) => {
   });
 };
 
+const gerUsersbyorganizationId = async organization_id => {
```
Comment (Swapna-Sahu) [inline src/server/api/controllers/user.controller.js]: Can you rename this "gerUsersbyorganizationId " to "getUsersByOrganizationId"

**Review by Swapna-Sahu — CHANGES_REQUESTED** [review]
Only one small change needed. Other than that all looks good to me.

File: src/server/api/controllers/user.controller.js (hunk 1 above)
Comment (ghost) [reply src/server/api/controllers/user.controller.js]: @Swapna-Sahu thanks for mentioning this...

File: src/server/api/controllers/user.controller.js [hunk 2]
```diff
@@ -321,6 +321,16 @@ const createAdminBySuperUser = async (id, body) => {
   });
 };
 
+const getUsersbyOrganizationId = async organization_id => {
+  const usersByOrganizationId = await knex
+    .from("users")
+    .where({ organization_id });
+  if (usersByOrganizationId.length === 0) {
+    return `This Organization have no users!`;
```
Comment (ismailirahim) [inline src/server/api/controllers/user.controller.js]: Throw an HTTP error here instead of just returning a string! :heart:

**Review by ismailirahim — CHANGES_REQUESTED** [review]
changes reuqest! :boom:

File: src/server/api/controllers/user.controller.js (hunk 2 above)
Comment (ghost) [reply src/server/api/controllers/user.controller.js]: Done ;)

**Review by AbodHassam — DISMISSED** [review]
Looks good to me

**Review by zkwsk — APPROVED** [review]
I pushed a small fix to swagger myself, but other than that it looks good 👍

**Pull request merged**